### DISCUSSION TITLE
PHPStorm license

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3818] [Team Posts]
@jekkos can you look to see if opensourcepos has any JetBrains licenses provisioned but not being used or being used by a dev who is no longer contributing to the project?  @owlbrudder could use PHPStorm and is willing to help the project quite a bit.

**Comment by jekkos** [comment]
Last time I renewed it I asked for three licenses. They have renewed mine and yours and replied the following

`Hello Jeroen,
 
Thanks for your reply.
 
We’ve extended 2 free All Products Pack licenses for another year. Please check your inbox for the new license certificate. When some new developer(s) join your project in the future, you could send us links to their activity — we'll be glad to review it.`

**Reply by jekkos** [reply]
So I can request it for him when I return back home nest week.
